ek9000App/src/devEK9000.h

```diff
@@ -218,6 +220,10 @@ class devEK9000 : public drvModbusAsyn {
 	/* Buffer for status info */
 	uint16_t m_status_buf[EK9000_STATUS_END - EK9000_STATUS_START + 1];
 
+	/* Cache of terminal layout */
+	uint16_t m_terminals[0xFF];
```

Comment:
Magic number `0xff` - yes, a well known one to be fair - should probably be a const/define somewhere?